recipes/gtest/all/conanfile.py

```diff
@@ -86,13 +86,16 @@ def package_id(self):
         del self.info.options.no_main
 
     def validate(self):
-        if self.options.shared and (is_msvc(self) or self._is_clang_cl) and "MT" in msvc_runtime_flag(self):
+        shared = self.info.options.shared
+        if (shared and (is_msvc(self) or self._is_clang_cl)
+                and "MT" in msvc_runtime_flag(self)):
```

Comment:
There's a new helper for this https://docs.conan.io/en/latest/reference/conanfile/tools/microsoft.html#conan-tools-microsoft-is-msvc-static-runtime

Should clean up + prevent the issue